Formula/e/espeak-ng.rb

```diff
@@ -0,0 +1,67 @@
+class EspeakNg < Formula
+  desc "WHATWG-compliant and fast URL parser written in modern C++"
```

Comment:
Did you forget to change this?

Reply:
yes, gonna update it now. good catch

Reply:
fix pr, https://github.com/Homebrew/homebrew-core/pull/170453